python_fundamentals-master/07_classes_objects_methods/07_01_car.py

```diff
@@ -10,20 +10,24 @@
 
 '''
 
-class car():
+class Car():
     def __init__(self, model, year, max_speed):
         self.model = model
         self.year = year
         self.max_speed = int(max_speed)
+    def speed_up(self):
+        self.speed_up = self.max_speed + 5
```

Comment:
Worth pointing out that this line, doesn't actually work more than once.
so I made a car and I did
`car.speed_up()`
`car.speed_up()`

It would be the same thing, max_speed + 5.  This isn't what we want. We want the second speed up to be max speed + 10.
How can you solve this?

Reply:
I'm not able to figure this out. I think there is actually a bug in my code... perhaps we need to discuss this together.